services/user.py

```diff
@@ -0,0 +1,59 @@
+from django.db import transaction
+
+from db.models import User
+
+
+@transaction.atomic
```

Comment:
why do you need transaction here?

Reply:
mentor give me this advise to use transaction decorator on this method

Reply:
@AlexPelevin99 my previous comment did not take into account the underlying logic of the function, thanks @viktoria-rybenchuk for pointing this out. Comment from @viktoria-rybenchuk makes sense because using the `transaction.atomic` makes more sense when there are transactions that involve several objects and the whole transaction makes sense only when all changes are committed to the DB, otherwise, it will fall back to an initial state. Here you perform an operation over only one newly created object, which is why explicitly declaring that this transaction should be atomic is too cautious. I will approve this PR, however, do not overuse this decorator and prefer it when transactions include multiple objects and they depend on each other.

Reply:
@Dimosphen1 @viktoria-rybenchuk ok, thank you, for your comments